rust/src/fvm/engine.rs

```diff
@@ -427,9 +427,6 @@ mod v3 {
                                         .unwrap_or(ErrorNumber::AssertionFailed),
                                 )))
                             }
-                            ExecutionEvent3::InvokeActor(cid) => {
-                                Some(ExecutionEvent::InvokeActor(cid))
```

Comment:
I'm removing this information from v3 traces. It's only used by the ETH JSON-RPC API, which is moderately broken at the moment.